### PR TITLE
docs(changelog): for v13.12.0 [skip ci]

### PR DESCRIPTION
**Pull request by tdeekens**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,40 @@
+## [13.12.0](https://github.com/commercetools/merchant-center-application-kit/compare/v13.11.0...v13.12.0) (2019-06-27)
+
+#### 🐛 Type: Bug
+
+- `jest-stylelint-runner`
```
Comment (montezume) [inline CHANGELOG.md]: shouldn't this be a feature?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,40 @@
...
+  - [#789](https://github.com/commercetools/merchant-center-application-kit/pull/789) fix(jest-stylelint-runner): include run file ([@montezume](https://github.com/montezume))
```
Comment (montezume) [inline CHANGELOG.md]: maybe also link to the original `jest-stylelint-runner` . commit? describing how it's new feature

Comment (tdeekens) [reply CHANGELOG.md]: Moved things around. 
35ab210

Comment (tdeekens) [reply CHANGELOG.md]: Taking a leap faith now :)

**Pull request merged**